lesson07/assignment/js/vue.js

```diff
@@ -0,0 +1,62 @@
+const SpringApp = {
+      data() {
+        return {
+          counter: 0,
+          imgSrc: "img/spring-melt-off.png",
+          imgAlt: "Girdwood slowly melts in May. We call this the mud season.",
+          imgSrc2: "img/spring-melt-off.jpg",
+          imgAlt2: "All the snowman melt away in the spring.",
+          imageShowing: true,
+          springtodos:[
+            {text: 'let snow melt'},
+            {text: 'scoop up old dog poo'},
+            {text: 'get in last minute spring skiing'},
+          ]
+
+        }
+      },
+      methods: {
+        toggleImage() {
+          if (this.imageShowing) {
+            this.imageShowing = false
+          } else {
+            this.imageShowing = true
+          }
+        }
+      }
+    }
+
+
+    Vue.createApp(SpringApp).mount('#spring')
+
+    const SummerApp = {
```

Comment:
Generally you will only want to have one Vue app. Though you are ahead of the game here in separating your code out. Vue components will make this super easy in later assignments :D